src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/ProgIdAttributeTests.cs

```diff
@@ -8,15 +8,15 @@ namespace System.Runtime.InteropServices.Tests
     [ProgId("pizza")]
     public class ProgIdAttributeTests
     {
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltInComEnabled))]
```

Comment:
I would expect this one to work with COM disabled - it's just reading an attribute.

Reply:
That is true, but the trimmer for AOT removes all COM related attributes. Since this attribute is only for COM related scenarios I was under the impression we could limit it to that.

Reply:
Ah, I see, you're right, this would be removed because we decided there's no other legitimate use for this attribute but COM and we strip it unconditionally.

The "correct" condition would be "COM is not enabled and the test was built with aggressive trimming" (there is a PlatformDetection for aggressive trimming), but conditioning on COM sounds sufficient since the test is low value anyway.